src/Helper/Scripts.php

```diff
@@ -116,36 +111,39 @@ public function addCondInternal($cond, $script, $pos = 100)
     /**
      * Adds internal script
      *
-     * @param int $pos The script position in the stack.
+     * @param int   $pos  The script position in the stack.
+     * @param array $attr The additional attributes
      *
-     * @return Scripts
+     *
+     * @return void
      *
      * @access public
      */
-    public function beginInternal($pos = 100)
+    public function beginInternal($pos = 100, array $attr = array())
     {
-        $this->capture[] = $pos;
+        $this->capture[] = array($pos, $attr);
          ob_start();
     }
 
     /**
-     * beginInternalCond
+     * Begin Conditional Internal Capture
      *
-     * @param mixed $cond DESCRIPTION
-     * @param int   $pos  DESCRIPTION
+     * @param mixed $cond condition
+     * @param int   $pos  position
+     * @param array $attr The additional attributes
      *
-     * @return mixed
+     * @return void
```

Comment:
Again, is void not correct for a method which returns nothing?
I've seen some debate in the past, but thought this was correct.
Do we prefer `null`? or omitting the `@return` tag? 
I've got my my syntax checking plugin in vim set to yell at me if I omit the `@return` on anything other than constructors, so I always just include it. I usual use `void` to differentiate between a function which never returns anything and a function which may explicitly `return null;`.